dino.icu.yaml

```diff
@@ -165,6 +165,11 @@ javier: # by https://github.com/J-cordz - my personal hack club scrapbook
    ttl: 600
    type: CNAME
    value: cname.vercel-dns.com.
+   
+jzaleta: # Javi's twitter redirect
+   ttl: 600
+   type: CNAME
+   value: https://x.com/_jzaleta.   
```

Comment:
Hey! DNS records can't have paths or protocols in them.

Reply:
Oops I totally missed that, already fixed!

Reply:
Hey, you still can't have paths in DNS records. I'd recommend figuring out how to redirect to your twitter profile. There should be a way to do it from whatever you're using to build your site.

Reply:
I figured out that the best workaround for this was to setup the dns record with a vercel site that makes the redirect. Hope it works now.

Reply:
> I figured out that the best workaround for this was to setup the dns record with a vercel site that makes the redirect. Hope it works now.

Vercel prefers setting up an A record over `cname.vercel-dns.com.` so did the change.